docs/FAQ/LightningNetwork.md

```diff
@@ -317,6 +317,77 @@ export BTCPAYGEN_ADDITIONAL_FRAGMENTS="$BTCPAYGEN_ADDITIONAL_FRAGMENTS;opt-lnd-c
 
 This way your custom settings gets added to the config and they will persist updates.
 
+### How to connect to a LND watchtower?
+
+To connect a LND watchtower you need to integrate the [`opt-lnd-wtclient`](https://github.com/btcpayserver/btcpayserver-docker/blob/master/docker-compose-generator/docker-fragments/opt-lnd-wtclient.yml) fragment and optionally configure the `LND_WTCLIENT_SWEEP_FEE`:
+
+```bash
+export BTCPAYGEN_ADDITIONAL_FRAGMENTS="$BTCPAYGEN_ADDITIONAL_FRAGMENTS;opt-lnd-wtclient"
+export LND_WTCLIENT_SWEEP_FEE=10 # Fee to be used for sweep transaction, 10 sat/vbyte is the default
+. ./btcpay-setup.sh -i
+```
+
+Afterwards you can use the `wtclient` RPC commands for managing the connections to watchtowers:
+
+```bash
+# Connect to a remote watchtower
+./bitcoin-lncli.sh wtclient add PUBKEY@IP:PORT
+
+# See your watchtower connections
+./bitcoin-lncli.sh wtclient towers
+```
+
+### How to run a LND watchtower?
+
+You can run a watchtower alongside your LND instance by integrating the [`opt-lnd-watchtower`](https://github.com/btcpayserver/btcpayserver-docker/blob/master/docker-compose-generator/docker-fragments/opt-lnd-watchtower.yml) fragment:
+
+```bash
+export BTCPAYGEN_ADDITIONAL_FRAGMENTS="$BTCPAYGEN_ADDITIONAL_FRAGMENTS;opt-lnd-watchtower"
+. ./btcpay-setup.sh -i
+```
+
+This will make the watchtower available on the server.
+
+To allow connections from other watchtower clients (via the [wtclient RPC commands]()), you will need to add the `watchtower.externalip` to a [custom fragment](../Docker/README.md#how-can-i-customize-the-generated-docker-compose-file) in `docker-compose-generator/docker-fragments/opt-lnd-config.custom.yml` like this:
+
+```yml
+version: '3'
+services:
+  lnd_bitcoin:
+    environment:
+      LND_EXTRA_ARGS: |
+        watchtower.externalip=YOUR_SERVER_IP
```

Comment:
I'm thinking if we should add this stuff in `opt-lnd-watchtower` , so if you enable watchtower fragment this is done for you as well.

I guess we need to merge documentation PR regardless, but point of improvement for future.

Reply:
I was hesitant to expose it publicly by default. Also it might be tricky if the server IP isn't static.